fe/example/src/widgets/CandidateList.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  Actions,
+  Badge,
+  Box,
+  Button,
+  Column,
+  Columns,
+  IconPromote,
+  Inline,
+  Stack,
+  Strong,
+  Text,
+} from 'braid-design-system';
+import React, { useState } from 'react';
+
+import { SearchField } from '../components/SearchField';
+import { Table, TableColumn } from '../components/Table';
+import { Candidate } from '../data/candidates';
+
+const ATTRIBUTE_ROW_LIMIT = 3;
+
+const COLUMNS: Array<TableColumn<Candidate>> = [
+  {
+    label: 'Name',
+    align: 'left',
+    render: (candidate) => candidate.formattedName,
+    to: (candidate) => `/candidates/detail/${candidate.id}`,
+    weight: 1,
+  },
+  {
+    label: 'Career history',
+    align: 'left',
+    render: (candidate) => (
+      <Stack space="small">
+        {candidate.roles.length === 0 ? (
+          <Text baseline={false} tone="secondary">
+            <IconPromote /> New to the workforce
+          </Text>
+        ) : (
+          candidate.roles.slice(0, ATTRIBUTE_ROW_LIMIT).map((role, index) => (
+            <Text baseline={false} key={index}>
+              {(() => {
+                if (index === 0) {
+                  return (
+                    <>
+                      <Strong>{role.title}</Strong>, {role.company}
+                    </>
+                  );
+                }
+
+                const extras = candidate.roles.length - ATTRIBUTE_ROW_LIMIT + 1;
+
+                if (index === ATTRIBUTE_ROW_LIMIT - 1 && extras > 1) {
+                  return <small>+{extras} others</small>;
+                }
+
+                return <small>{role.title}</small>;
+              })()}
+            </Text>
+          ))
+        )}
+      </Stack>
+    ),
+    weight: 3,
+  },
+  {
+    label: 'Education',
+    align: 'left',
+    render: (candidate) => (
+      <Stack space="small">
+        {candidate.qualifications
+          .slice(0, ATTRIBUTE_ROW_LIMIT)
+          .map((qualification, index) => (
+            <Text baseline={false} key={index}>
+              {(() => {
+                if (index === 0) {
+                  return (
+                    <>
```

Comment:
How am I just learning about this?

Reply:
Oof that's inconsistent. @benjervis vetoed it like a year ago which is why it's seen nowhere else in our code.